Sources/Sentry/include/SentryHub.h

```diff
@@ -15,6 +15,7 @@ SENTRY_NO_INIT
 // Since there's no scope stack, single hub instance, experimenting with holding
 // session here.
 @property (nonatomic, readonly, strong) SentrySession *_Nullable session;
+@property (nonatomic, readonly, strong) NSNumber *isEnabled;
```

Comment:
Any reason why this is an `NSNumber*` and not a `BOOL`; ie. type differs from `SentrySDK.isEnabled`?